{
  "language": "java",
  "id": "commons-lang-DefaultExceptionContext_129",
  "buggyPath": ".",
  "referencePath": null,
  "buildCommand": "mvn package -V -B -Denforcer.skip=true -Dcheckstyle.skip=true -Dcobertura.skip=true -Drat.skip=true -Dlicense.skip=true -Dfindbugs.skip=true -Dgpg.skip=true -Dskip.npm=true -Dskip.gulp=true -Dskip.bower=true -Drat.numUnapprovedLicenses=100 -DskipTests=true -DskipITs=true -Dtest=None -DfailIfNoTests=false",
  "testCommand": "mvn test -V -B -Denforcer.skip=true -Dcheckstyle.skip=true -Dcobertura.skip=true -Drat.skip=true -Dlicense.skip=true -Dfindbugs.skip=true -Dgpg.skip=true -Dskip.npm=true -Dskip.gulp=true -Dskip.bower=true -Drat.numUnapprovedLicenses=100",
  "categories": [
    "safety",
    "npe"
  ],
  "npe": {
    "filepath": "src/main/java/org/apache/commons/lang3/exception/DefaultExceptionContext.java",
    "line": 138,
    "npe_method": "getFormattedExceptionMessage",
    "deref_field": "baseMessage",
    "npe_class": "DefaultExceptionContext",
    "repo": "commons-lang",
    "bug_id": "DefaultExceptionContext_129"
  }
}
